test-tsc.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Google LLC
```

Comment:
Let’s just import test from examples dir. see other tests.

Reply:
I suggest reusing `test.mjs` instead.

Reply:
How do you like this trick?